bitwarden_license/src/app/organizations/components/input-text.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, Input, OnInit, Self } from "@angular/core";
+
+import { BaseCvaComponent } from "./base-cva.component";
+
+@Component({
```

Comment:
Let's add a disclaimer comment on these as well to not use them outside the SSO config context.